### PR TITLE
refactor: improvement the testing of ticketing system CRUD

### PR DESCRIPTION
**Pull request by Baatarvant**

### CODE REVIEW

**Comment by billduluu** [comment]
@Baatarvant conflittoi bn

File: backend/src/controller/ticketControllers.ts [hunk 1]
```diff
@@ -22,31 +15,40 @@ export const getTicketService = async (request: Request, response: Response) =>
...
     response.status(200).send(result);
   } catch (error) {
-    /* istanbul ignore next */
-    response.status(500).send(error);
+    response.status(400).send(error);
   }
 };
 
 export const createTicketService = async (request: Request, response: Response) => {
+  if (Object.keys(request.body).length === 0) {
```
Comment (Narantsatsralt05) [inline backend/src/controller/ticketControllers.ts:44]: I think _.isEmpty() it is better to check whether it is empty or not 😃

File: backend/tests/routes/ticketingService.test.ts [hunk 2]
```diff
@@ -18,81 +18,113 @@ const newTicket = {
   status: "open",
   type: "office",
   priority: "low",
-  description: "test description",
+  description: "description",
 };
```
Comment (Narantsatsralt05) [inline backend/tests/routes/ticketingService.test.ts:23]: header havidaa esvel import iin door 

```js
// 1. iim test negt coverloson
// 2. iim test hoyrt coverloson 
```

gevel unshihad amar bna

File: backend/src/controller/ticketControllers.ts (hunk 1 above)
Comment (Baatarvant) [reply backend/src/controller/ticketControllers.ts:44]: There is no isEmpty() method in Javascript.

**Review by Narantsatsralt05 — APPROVED** [review]

**Pull request merged**